docs/docs/how-tos/branching.ipynb

```diff
@@ -192,6 +201,16 @@
     "graph.invoke({\"aggregate\": []}, {\"configurable\": {\"thread_id\": \"foo\"}})"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "ea5495cf-9564-40c6-bc2d-0b2a8f72a5df",
+   "metadata": {},
+   "source": [
+    "!!! note\n",
+    "\n",
+    "    In the above example, nodes `\"b\"` and `\"c\"` are executed concurrently in the same [superstep](../../concepts/low_level/#graphs). Because they are in the same step, node `\"d\"` executes after both `\"b\"` and `\"c\"` are finished."
+   ]
+  },
```

Comment:
the callout below needs to be clarified actually: while the whole superstep errors, we DO actually save the results from successful nodes (when used w/ a checkpointer) so that when we resume, we don't re-execute those nodes